### ISSUE TITLE
setuptools(>58) no longer supports 2to3

### ISSUE THREAD

**Issue by cmoeckel91** [keflavich/image_registration#34]
I recently updated my pip base functionalities, including setuptools (59) and failed to install image_registration. I had to revert to setuptools 57.5.0 to install it using pip. 

pip install setuptools==57.50

**Comment by back2yes** [comment]
Just met up with same problem here. Worked around it by changing the value of `use_2to3` in `setup.py` from `True` to `False`

```
setup(name=PACKAGENAME,
      version=VERSION,
      description=DESCRIPTION,
      scripts=scripts,
      install_requires=REQUIREMENTS,
      author=AUTHOR,
      author_email=AUTHOR_EMAIL,
      license=LICENSE,
      url=URL,
      long_description=LONG_DESCRIPTION,
      cmdclass=cmdclassd,
      zip_safe=False,
      use_2to3=False,  # change this from `True` to `False`
      entry_points=entry_points,
      **package_info
)
```

**Comment by keflavich** [comment]
want to submit that fix as a PR?

**Comment by back2yes** [comment]
> want to submit that fix as a PR?

Sure.

**Comment by keflavich** [comment]
Closed by #35 , thanks @back2yes !
